Sources/MongoSwift/BSON/Document.swift

```diff
@@ -213,13 +218,11 @@ public struct Document: ExpressibleByDictionaryLiteral, ExpressibleByArrayLitera
      *  let x: Int = try d.get("x")
      *  ```
      *
-     *  - Params:
+     *  - Parameters:
```

Comment:
Did you plan on making this change across the project, or are most other doc blocks already using "Parameters" over "Params"?